easybuild/easyconfigs/g/geopandas/geopandas-1.0.1-foss-2023b.eb

```
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'geopandas'
+version = '1.0.1'
+
+homepage = 'https://geopandas.org'
+description = """GeoPandas is a project to add support for geographic data to pandas objects.
+It currently implements GeoSeries and GeoDataFrame types which are subclasses of pandas.Series
+and pandas.DataFrame respectively. GeoPandas objects can act on shapely geometry objects and
+perform geometric operations."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+builddependencies = [
+    ('Cython', '3.0.10'),  # required for pyogrio
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('SciPy-bundle', '2023.11'),
+    ('Shapely', '2.0.6'),
+    ('Fiona', '1.10.1'),
+    ('pyproj', '3.6.1'),
+    ('matplotlib', '3.8.2'),  # optional
+    ('psycopg2', '2.9.10'),  # optional
+    ('SQLAlchemy', '2.0.29'),  # optional
+    ('networkx', '3.2.1'),  # needed by mapclassify
+    ('scikit-learn', '1.4.0'),  # needed by mapclassify
+    ('numba', '0.60.0'),  # optional, numba extra in mapclassify
+]
+
+use_pip = True
+
+exts_list = [
+    ('pyogrio', '0.10.0', {
+        'checksums': ['ec051cb568324de878828fae96379b71858933413e185148acb6c162851ab23c'],
+    }),
+    ('mapclassify', '2.8.1', {
+        'checksums': ['306f4cb99ad1ea166b3efd7180c0a199d240bd801de7937327973d829673bc82'],
+        'use_pip_extras': 'numba',
+    }),
+    (name, version, {
+        'checksums': ['6e71d57b8376f9fdc9f1c3aa3170e7e420e91778de854f51013ae66fd371ccdb'],
```

this still had the checksum of the older version
```suggestion
        'checksums': ['b8bf70a5534588205b7a56646e2082fb1de9a03599651b3d80c99ea4c2ca08ab'],
```